WordPress/Classes/Utility/ContextManager+ErrorHandling.swift

```diff
@@ -55,8 +55,38 @@ private extension NSExceptionName {
     static let coreDataSaveDerivedException = NSExceptionName("Unresolved Core Data save error (Derived Context)")
 }
 
-extension ContextManager {
-    @objc(handleSaveError:inContext:)
+extension LegacyContextFactory {
+
+    /// A wrapper of `internalSave(_:)` to expose it as an Objective-C API for `LegacyContextFactory` to call.
+    @objc func internalSaveContext(_ context: NSManagedObjectContext) {
```

Comment:
Adding `@objc` to the `internalSave(_:)` function doesn't compile, since we can't add a "category method" to Objective-C protocol. I have to add this function for Objective-C class `LegacyContextFactory`.